src/firewall-cmd.in

```diff
@@ -1152,6 +1146,20 @@ except FirewallError as msg:
     cmd.print_and_exit("Error: %s" % msg, code)
 
 fw.setExceptionHandler(cmd.exception_handler)
+
+if not (options_standalone or options_ipset or \
+        options_icmptype or options_service or options_helper or \
+        options_config or options_zone_ops or options_policy or \
+        options_direct or options_permanent_only):
+    state = fw.get_property("state")
+    if state == "RUNNING":
+        state = "State: running\n\n"
+    elif state == "FAILED":
+        state = "State: failed\n\n"
+    else:
+        state = "State: not running\n\n"
+    cmd.fail(state + "No options specified\n" + parser.format_usage()[:-1])
```

Comment:
What is the `[:-1]` string slicing for?

Reply:
There's an extra new-line that I had thought looked strange that appeared to be added a part of format_usage(), so the slicing just removes that last character.

Reply:
It's fine as is. Thanks.